utils/src/logger.ts

```diff
@@ -22,8 +22,6 @@ const logs = new Map<Source, Set<FileLog>>();
 
 process.once('beforeExit', emitLogs);
 
-/* node:coverage disable */
-// this is actually coveraged, but v8 doesn't recognise it
```

Comment:
Is this no-longer the case?

Reply:
![Uploading Capture d’écran 2025-06-29 à 23.55.31.png…]()

Reply:
<img width="655" alt="Capture d’écran 2025-06-29 à 23 55 31" src="https://github.com/user-attachments/assets/25dbae06-fdbb-46a1-9c0f-c5b9bcc7b223" />